<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialSettings</class>
 <widget class="QWidget" name="serialSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>164</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>164</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>193</width>
    <height>164</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="portlabel">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>49</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial Port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitsCombo">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>84</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of stop bits per symbol. This is almost always 1.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="portName">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>6</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The serial port to which the system is connected. All ports listed here should work, unless marked INVALID (this can happen if a port has been removed from the system after this dialog was first used).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="minimumContentsLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="portlabel_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>32</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Baud Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="portlabel_5">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>136</y>
     <width>43</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data bits</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBitsCombo">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>136</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of data bits per symbol. This is almost always 8.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="portlabel_6">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>84</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop bits</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRate">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>32</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The data transmission rate. If unsure 57600 and 115200 are very common rates.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>75</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>134</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>56000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>76800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>460800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>921600</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Parity">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>110</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_flowControl">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>58</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Flow Control</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_flowControl">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>58</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the flow control type. In most cases this is None or HW (aka. DTS/RTS). SW type is aka. XON/XOFF.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SW</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Parity">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>110</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the parity. In most cases no parity (None) is used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
  </widget>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete this link</string>
   </property>
   <property name="statusTip">
    <string>Delete this link</string>
   </property>
   <property name="whatsThis">
    <string>Link delete button</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect this link</string>
   </property>
   <property name="statusTip">
    <string>Connect this link</string>
   </property>
   <property name="whatsThis">
    <string>Connect this link</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close the configuration window</string>
   </property>
   <property name="statusTip">
    <string>Close the configuration window</string>
   </property>
   <property name="whatsThis">
    <string>Close the configuration window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>serialSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
